source/settings.cpp

```diff
@@ -215,7 +215,7 @@ Option ShuffleRewards         = Option::U8  ("Shuffle Dungeon Rewards",{"End of
 Option LinksPocketItem        = Option::U8  ("Link's Pocket",          {"Dungeon Reward", "Advancement", "Anything", "Nothing"},          {linksPocketDungeonReward, linksPocketAdvancement, linksPocketAnything, linksPocketNothing});
 Option ShuffleSongs           = Option::U8  ("Shuffle Songs",          {"Song Locations", "Dungeon Rewards", "Anywhere"},                 {songsSongLocations, songsDungeonRewards, songsAllLocations});
 Option Shopsanity             = Option::U8  ("Shopsanity",             {MultiVecOpts({{"Off"}, NumOpts(0, 4), {"Random"}})},              {shopsOff, shopsZero, shopsOne, shopsTwo, shopsThree, shopsFour, shopsRandom});
-Option ShopsanityPrices       = Option::U8  (2, "Shopsanity Prices",   {"Random", "Affordable"},                                          {shopPriceRandom, shopPriceAffordable});
+Option ShopsanityPrices       = Option::U8  (2, "Shopsanity Prices",   {"Random","Affordable","Child wallet","Adult wallet", "Giant's wallet","Tycoon wallet"},                               {shopPriceRandom, shopPriceAffordable,shopPriceChild,shopPriceAdult,shopPriceGiant,shopPriceTycoon});
```

Comment:
I think indenting like this looks better
```suggestion
Option ShopsanityPrices       = Option::U8  (2, "Shopsanity Prices",   {"Random", "Affordable", "Child wallet", "Adult wallet",
                                                                        "Giant's wallet", "Tycoon wallet"},                               {shopPriceRandom, shopPriceAffordable, shopPriceChild, shopPriceAdult, shopPriceGiant, shopPriceTycoon});
```